tests/load/sources/filesystem/test_filesystem_source.py

```diff
@@ -300,6 +300,32 @@ def _get_files() -> DltResource:
     assert pipeline.last_trace.last_normalize_info.row_counts["csv_files_2"] == 4
 
 
+@pytest.mark.parametrize(
+    "bucket_url",
+    [
+        HTTP_BUCKET,
+    ],
+)
+@pytest.mark.parametrize(
+    "destination_config",
+    destinations_configs(default_sql_configs=True),
+    ids=lambda x: x.name,
+)
+def test_http_filesystem(
+    public_http_server, bucket_url: str, destination_config: DestinationTestConfiguration
+):
+    public_resource = filesystem(bucket_url=bucket_url, file_glob="parquet/mlb_players.parquet")
```

Comment:
does globbing work now? or this is still single file?

Reply:
> if we change anything that is user facing then we always improve documentation

That makes perfect sense,  the docs should follow!

My suggestion for the documentation came directly from talking with a developer who uses dlt in production. I noticed that using S3-compatible storage felt a bit different from hitting a direct HTTP endpoint. Even working with a standard REST API is not very close to this case. 

Basically, my thinking is that the developer experience for this new feature is subtle, and the case is not immediately obvious. I am not trying to push the idea, rather to share the reasoning behind the suggestion.